build.rs

```diff
@@ -84,6 +86,26 @@ fn main() {
     if linux_time_bits64 {
         set_cfg("linux_time_bits64");
     }
+    println!("cargo:rerun-if-env-changed=RUST_LIBC_UNSTABLE_GNU_FILE_OFFSET_BITS64");
+    match env::var("RUST_LIBC_UNSTABLE_GNU_FILE_OFFSET_BITS64") {
+        Ok(val) => {
+            if val == "1" {
```

Comment:
```suggestion
        Ok(val) => if val == "1" {
```

Nit to remove an indent level. The next arm needs to change to `Ok(_v) | Err(_e)`.

Reply:
Done